src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+
+![Ui screens](images/IBM_UI_Screens.gif)
+
+
+## Color anatomy
+
+IBM Product Design uses a subset of the IBM Design Language color palette. The Neutral Gray family is dominant in our UI, making use of subtle shifts in value to help organize content into distinct zones. 
+
+The core Blue family has been designated as the primary action color across all products and experiences, ensuring our color aesthetic is a part of every interaction. Additional colors are used sparingly and purposefully.
+
+![Color anatomy](images/color_anatomy_bkg.svg)
+
+### Color terms
+
+| Term      | Definition                                                                                                  |
+| --------- | ----------------------------------------------------------------------------------------------------------- |
+| Theme | The set of unique values assigned to each token.                                                            |
+| Token | The code identifier for a unique role or set of roles. Tokens are universal and never change across themes. |
+| Role | The systematic usage/s of a value. Roles cannot be changed between themes.                                  |
+| Value | The actual style (ie hex code) assigned to a token.                         |
+
+
+## UI Color themes
+
+Our palette is designed and implemented in a themable manner. Each theme works with a specific set of primary background colors. These colors are global variables which determine the color of the component being applied within each theme. The light theme uses White and Gray 10 backgrounds and the dark theme uses Gray 100, 90 and 80 backgrounds. Default color tokens are provided for each component based on the primary background color.
+
+
+
+| Theme | Primary Background       | Token                | Hex value    |    |
+| --- | ----------- | ------------------- | -------------------- | ------------------- |
+| White   | Global Background Light  | `$ui-background`        | <color-block showhex="true" size="xs">#ffffff</color-block>  |
+| Gray 10   | Global Background Light | `$ui-background`  | <color-block showhex="true" size="xs">#f3f3f3</color-block>  |
+| Gray 90  | Global Background Dark   | `$ui-background` | <color-block showhex="true" size="xs">#282828</color-block>    |
+| Gray 100   | Global Background Dark  | `$ui-background`        | <color-block showhex="true" size="xs">#171717</color-block>     |
+
+
+
+
+### Tokens
+
+These themes are implemented with color tokens. In order to tokenize, a set of universal color variables must be determined by common roles and usage; tokens are not based singularly on a color value (i.e. unique hex code). The same color value may be assigned to multiple variables in a theme’s palette when the values have distinctly different roles.
```

Comment:
This isn't quite right either:  "In order to tokenize, a set of universal color variables must be determined by common roles and usage." Tokens are variables, so this sounds a little inception-y. Rewrite below:
"Tokens are created by determining common color roles and usages throughout the system."

Also the second half of the semi-colon section goes more with the last sentence. Rewrite below:
"Tokens are not based singularly on a color value (i.e. unique hex code); the same color value may be assigned to multiple variables in a theme’s palette if the values have distinctly different roles. **For example, in the White theme the color white is used for both text on a button as well as primary background. The white for text is assigned the token `$text-04` and the white for background is `$ui-background`**."

I added in that last bolded part.